phpunit/phpunit/4.7/phpunit.xml.dist

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 
-<!-- https://phpunit.de/manual/current/en/appendixes.configuration.html -->
+<!-- https://phpunit.readthedocs.io/en/latest/configuration.html -->
 <phpunit xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
          xsi:noNamespaceSchemaLocation="http://schema.phpunit.de/6.5/phpunit.xsd"
```

Comment:
this should also be changed to https as http just redirects to https

Reply:
I'm not sure about it. It's reported as is on every configuration in phpunit repository

Reply:
Hm, probably should be fixed upstream as well?